src/integrationtests/resources/features/FailedNotificationIT.feature

```diff
@@ -0,0 +1,44 @@
+#Author: christoffer.cortes.sjowall@ericsson.com
+
+Feature: Failed Notification Integrationtest
+
+  Scenario:
+    Start an artifact flow aggregation and make sure the email notification fails
+    due to incorrect SMTP server port. The port is set with @TestPropertySource
+    in the steps file.
+
+    # Setup Eiffel Intelligence
+    Given the rules "src/main/resources/rules/ArtifactRules-Eiffel-Agen-Version.json"
+    And the events "src/test/resources/ArtifactFlowTestEvents.json"
+
+    # Setup subscription
+    Given subscription object for "MAIL" with name "MailFailureTestSubscription" is created
```

Comment:
for "MAIL" doesn't really read well. I had to go to the step file to see what it actually meant. I think phrasing it like: "subscription object of type "MAIL" with name ... "reads better.

Reply:
Done. I'll change it in the old feature files as well since this was essentially a copy paste of those.